app/controllers/game_nights_controller.rb

```diff
@@ -0,0 +1,55 @@
+class GameNightsController < AuthenticationController
+  def create
+    result = CreateModel.call(active_record_class: GameNight,
```

Comment:
@Enriikke can you think of a better way to extract the class? This is a little inelegant :smile_cat: